pkg/yang/types.go

```diff
@@ -139,6 +139,16 @@ func (t *Typedef) resolve() []error {
 		y.Default = t.Default.Name
 	}
 
+	if t.Type.IdentityBase != nil {
+		// We need to copy over the IdentityBase statement if the type has one
+		if idBase, err := RootNode(t).findIdentityBase(t.Type.IdentityBase.Name); err == nil {
+			y.IdentityBase = idBase.Identity
+		} else {
+			return []error{fmt.Errorf("Could not resolve identity base for typedef: %s",
+				t.Type.IdentityBase.Name)}
```

Comment:
Join this with the line above.

I used to always indent my parameters, but Go style seems to be "make a long line and use shorter names".   Even though I preferred idents in the past, now it seems easier to read without them, mainly because they are rarely used (there still are times to indent parameters, but this one is short so no reason).